charts/greptimedb-standalone/values.yaml

```diff
@@ -35,6 +35,10 @@ args: []
 # -- Environment variables
 env:
   GREPTIMEDB_STANDALONE__HTTP__ADDR: "0.0.0.0:4000"
+  GREPTIMEDB_STANDALONE__GRPC__ADDR: "0.0.0.0:4001"
```

Comment:
suggestion: Can we define the configuration field instead of using environment variables? Environment variables are not a common way to configure db.

Reply:
I will find a good way to configure these environment variables. But now open the http port, because the monitor needs this configuration.